source/NVDAObjects/UIA/__init__.py

```diff
@@ -1182,17 +1182,29 @@ def findOverlayClasses(self,clsList):
 			from . import VisualStudio
 			VisualStudio.findExtraOverlayClasses(self, clsList)
 
-		# Support Windows Console's UIA interface
+		# Support Windows Console and Terminal
+		_all_wt_UIAClassNames = frozenset((

```

Comment:
I think typical syntax for this is, but I don't think it matters
```suggestion
		_all_wt_UIAClassNames = frozenset({
```